src/website/blog/2025-12-04-newsletter.md

```diff
@@ -0,0 +1,141 @@
+  relevant in the age of AI? ([Emmanuel Paraskakis]()).
+
+2026 promises to be an exciting year and in bringing 2025 to close our OAI
+Ambassador and custodian of the OAI Track Erik Wilde puts the outlook for future
+events like this: "_For the past two years, OAI has actively fostered the
+OpenAPI community with organizing events at various events, most recently with
+our first OpenAPI Conference at API Days Paris. We plan on continuing these
+efforts in 2026, tentatively planning events in San Jose, Singapore, New York,
+London, Santa Clara, and Paris. If you're interested in APIs and OpenAPI, join
+us at one of these events in 2026! We also just launched our very own
+[conference site](https://conference.openapis.org/) so that going forward, you
+can find up-to-date information about all of our events in one place._"
+
+Our new conference and events website has been created to help highlight the
+work that OAI is doing bringing in-person events to the community, and will
+provide a complete picture of the agendas, talk and workshop abstracts, and
+speaker profiles in one place. Massive thanks here to
+[Pavel Kornev](https://www.linkedin.com/in/pavelkornev/), from OAI member SAP,
+who led the project with a clear vision for a cleaner, more modern experience,
+with [Juri Jatschmenow's](https://www.linkedin.com/in/jurijat/) outstanding
+development work bringing that vision to life. On his teams contributions, Pavel
+says: "_Designing the new OpenAPI Conference Paris 2025 landing page has been an
+exciting journey for our team. We’re both proud to contribute to the OpenAPI
+Initiative and thrilled to support an event that brings the community together.
+This is just the beginning — we’re looking forward to enhancing even more
+landing pages across the Initiative._"
+
+### Initiative News
+
+There’s been a great reception to our OpenAPI Specification v3.2 release, with a
+great deal of coverage and interest from the community in the new features and
+capabilities. As a reminder, v3.2 provides a host of changes such as the
+refactored and improved [Tag Object](), support for the `QUERY` HTTP Method,
+support for sequential and streaming data protocols, and additional Security
+Scheme features like OAuth 2.0 Device Authorization Flow. You can find out more
+in our
+[blog post](https://www.openapis.org/blog/2025/09/23/announcing-openapi-v3-2)),
+which includes links to key resource to help with upgrading to v3.2. Our
+[Ecosystem Spotlight](#ecosystem-spotlight) also provides resources from the
+community on what the v3.2 upgrade means.
+
+Completing v3.2 means we are now looking at what could appear in our next
+version and beyond, and the team is ramping up for further releases in 2026,
+with an expected v3.3, covering improvements to Security Schemes and greater
+integration with MCP and AI protocols more generally.
+
+[Henry Andrews](https://github.com/handrews) has already pulled together a
```

Comment:
This should be framed as one contributor's suggestions for our next few releases, with contributions from other folks expected.  There are some things I'm pretty sure we'll do (for example because @karenetheridge is looking into them) that I didn't include.  I just tend to post grand plans more than other folks.  I focused on things that I thought might need more attention rather than projects I thought already had people attached to them.

Reply:
I post grand plans but nobody realises they are grand